contrib/depends/hosts/android.mk

```diff
@@ -0,0 +1,20 @@
+ANDROID_API=21
+
+ifeq ($(host_arch),arm)
+host_toolchain=arm-linux-androideabi-
+endif
+
+android_CC=$(host_toolchain)clang
+android_CXX=$(host_toolchain)clang++
+
+android_CFLAGS=-pipe
+android_CXXFLAGS=$(android_CFLAGS)
+
+android_release_CFLAGS=-O2
+android_release_CXXFLAGS=$(android_release_CFLAGS)
+
+android_debug_CFLAGS=-O1
```

Comment:
-g ?

Reply:
Oh. probably a good idea, yeah. Does anyone ever do depends builds in Debug mode? Will fix.